core/azure-core/src/samples/java/com/azure/core/util/polling/PollResponseJavaDocCodeSnippets.java

```diff
@@ -25,7 +25,7 @@ public void initialise(String otherStatus, T value) {
         // BEGIN: com.azure.core.util.polling.pollresponse.custom.status
         // Lets say we want to crete poll response with status as IN_PROGRESS
         PollResponse<String> inProgressPollResponse
-            = new PollResponse<>("CUSTOM_OTHER_STATUS", "my custom response");
+            = new PollResponse<>(PollResponse.OperationStatus.fromString("CUSTOM_OTHER_STATUS"), "my custom response");
```

Comment:
Do a static import of OperationStatus so you don't need to fully-qualify the `PollResponse.` part.